toolkits/golang.go

```diff
@@ -337,16 +356,23 @@ func (toolkit GoToolkit) PrepareForStepRun(step stepmanModels.StepModel, sIDData
 	}
 
 	// it's not cached, so compile it
-
 	if step.Toolkit == nil {
 		return errors.New("No Toolkit information specified in step")
 	}
 	if step.Toolkit.Go == nil {
 		return errors.New("No Toolkit.Go information specified in step")
 	}
-	packageName := step.Toolkit.Go.PackageName
 
-	return goBuildInIsolation(packageName, stepAbsDirPath, fullStepBinPath)
+	isInstallRequired, _, goConfig, err := selectGoConfiguration()
```

Comment:
Could you please elaborate on the reason why did you move it here?

Reply:
Wanted to test goBuildStep and the go path on the actual machine would have affected the command run, this was a cheap fix for that.